src/Plugin/Autocomplete/Handler.php

```diff
@@ -71,6 +73,29 @@ public function run(): Task {
 		return $task->run();
 	}
 
+	/**
+	 * Perform validation that is should be run in coroutine
```

Comment:
Has this comment got copied from another file?

Reply:
Changed a bit for clarification